widget/form_test.go

```diff
@@ -121,17 +121,39 @@ func TestForm_ChangeTheme(t *testing.T) {
 		},
 		OnSubmit: func() {}, OnCancel: func() {}}
 	w := test.NewWindow(form)
-	w.Resize(fyne.NewSize(340, 240))
 	defer w.Close()
 
 	test.AssertImageMatches(t, "form/theme_initial.png", w.Canvas().Capture())
 
 	test.WithTestTheme(t, func() {
 		form.Refresh()
+		w.Resize(form.MinSize().Add(fyne.NewSize(theme.Padding()*2, theme.Padding()*2)))
```

Comment:
Would it be better to use `w := test.NewWindow(layout.NewPaddedLayout(form))`?

Reply:
Unfortunately not - this is accounting for window padding. If you add more content it's still off by padding amount.

Reply:
Hmm, windows don't include this padding when ensuring they are big enough for their content?

Reply:
This is the crux of the matter with tests and internally-size-adjusting content. There is no "ensure the window is big enough for new content" outside of requesting it manually. Previously it was manual numbers, now it is calculated.

Reply:
Ack, something for another PR